test.sh

```diff
@@ -1,5 +1,67 @@
 #!/bin/bash
 
 # main test runner for heketi
+# Executes all executable scripts under tests dir
+# in sorted order.
 
-# TODO: add tests here...
+vecho () {
```

Comment:
So this whole thing is fun, but I think it should be in test/test-whatever.sh instead. And call that script from here in test.sh